### PR TITLE
minor code cleanup

### PR DESCRIPTION
**Pull request by PariyaAshok**

### CODE REVIEW

File: upi/openstack/compute-nodes.yaml [hunk 1]
```diff
@@ -86,5 +81,6 @@
       userdata: "{{ lookup('file', 'worker.ign') | string }}"
       nics:
       - port-name: "{{ os_port_worker }}-{{ item.0 }}"
+      meta: "{{ cluster_id_tag }}"
```
Comment (snaiksat) [inline upi/openstack/compute-nodes.yaml:84]: Are we should that this should go here and not above L71?

Comment (PariyaAshok) [reply upi/openstack/compute-nodes.yaml:84]: Yes , It should go to L71. Will add that.

**Review by abhis2112 — APPROVED** [review]

**Comment by snaiksat** [comment]
@PariyaAshok could you please let us know if the new install succeeds with this branch, and if so, then we can merge the PR.

**Comment by PariyaAshok** [comment]
> @PariyaAshok could you please let us know if the new install succeeds with this branch, and if so, then we can merge the PR.

We can merge the PR.

**Review by snaiksat — APPROVED** [review]

**Pull request merged**